go/vt/schema/ddl_strategy.go

```diff
@@ -209,7 +209,7 @@ func (setting *DDLStrategySetting) IsPreferInstantDDL() bool {
 
 // IsFastRangeRotationFlag checks if strategy options include --fast-range-rotation
 func (setting *DDLStrategySetting) IsFastRangeRotationFlag() bool {
-	return setting.hasFlag(fastRangeRotationFlag)
+	return true
```

Comment:
Do we want to clean up this function in a follow up PR?

Reply:
Function is now removed.